tests/plugins/editorplaceholder/editorplaceholder.js

```diff
@@ -205,5 +205,20 @@
 		} );
 	};
 
+	// (#4253)
+	tests[ 'test placeholder loads correctly in full-page editor with bbcode plugin'] = function() {
```

Comment:
Well, this test fails without the fix, which is good, but not in a way it should:

![](https://user-images.githubusercontent.com/1061942/98699327-4fb0cf80-2377-11eb-89ba-32fdb656b11e.png)

The correct way would be to `spy/stub` console error with `sinon` to see if no errors where thrown - if error is thrown use `assert.fail( ... )` to fail the test and if editor fires [`instanceReady` event](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#event-instanceReady) test should be marked as passing with `assert.pass( ... )`.

Reply:
Test changed with: https://github.com/ckeditor/ckeditor4/pull/4370/commits/742a64a46273fcf5caad5667764349e1b5a81f28

But for RunTime errors there is no `console.error` invokes. It looks like it in dev-console, but has no effect in code.
Finally I bind with `window.addEventListener( 'error', onErrorStub );` to catch RunTime error. Also checks for common `console.error` in case of other fails in future.

Reply:
Ok, makes sense :+1: